.rubocop.yml

```diff
@@ -49,6 +49,7 @@ Metrics/BlockLength:
 Metrics/ModuleLength:
   Exclude:
     - '**/*_spec.rb'
+    - 'lib/active_data.rb'
```

Comment:
Not sure how to handle it, all typecaster are already longer than threshold & moving them to the separate files [per class] is an overkill.